controllers/apiv3/api_base_controller.py

```diff
@@ -91,7 +91,7 @@ def post(self, *args, **kw):
         self._validate_tba_auth_key()
         self._errors = ValidationHelper.validate_request(self)
         if self._errors:
-            self.abort(400)
+            self.abort(404)
```

Comment:
@phil-lopreiato @fangeugene [`validate_request`](https://github.com/the-blue-alliance/the-blue-alliance/blob/master/helpers/validation_helper.py#L36-L98) is all about checking whether or not a key exists so this should be fine.